### PR TITLE
Move to pulling sermons from Sanity data source

### PR DESCRIPTION
**Pull request by tomduckering**
Moving to use Sanity as a data source as:

1) Graphcool is going away
2) Sanity provides nice UI and API for data editing and upload.

### CODE REVIEW

File: src/components/App/index.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ class App extends Component<Props, State> {
     render() {
         const { selectedSeriesId, talksFilter } = this.state;
         return (
-            <WithSerieses>
+            <WithSeriesesFromSanity>
                 {({ loading, error, serieses }) => {
                     if (loading === true || error != null) {
```
Comment (mauriceyap) [inline src/components/App/index.js:115]: not your change, but
```typescript
if (loading || error) {
    ...
}
```

File: src/components/WithSerieses/sanityQuery.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import sanityClient from "@sanity/client";
+import React, { Component, useState, useEffect } from "react";
+
+const ccmSanityClient = sanityClient({
+  projectId: "ip162aeb",
+  dataset: "production",
+  useCdn: true
+});
+
+const useQuerySanity = (query = "") => {
```
[the remark this answers is not in the document]
Comment (tomduckering) [reply src/components/WithSerieses/sanityQuery.js:10]: agreed.

File: src/components/WithSerieses/index.js [hunk 3]
```diff
@@ -77,3 +79,46 @@ class WithSerieses extends Component<Props> {
...
+                    return children({
+                        serieses: sortSerieses(serieses),
+                        loading,
+                        error: error != null ? error.message : null,
+                    });
+                }}
+            </SanityQuery>
+        );
+    }
+}
```
[the remark this answers is not in the document]
Comment (tomduckering) [reply src/components/WithSerieses/index.js:124]: Cool - annoyingly this isn't a typescript project. It's JS + Flow. I tried to convert it to Typescript but it ended up being a pain.

**Comment by tomduckering** [comment]
@mauriceyap - gonna skip your (good) bigger suggestion as it's not a TS app and I want to avoid some pain.

**Pull request merged**